repo/js/CD-Aware-AutoGather/README.md

```
@@ -12,46 +23,150 @@
+### 队伍配置推荐
 
 采集任务可能用到的元素共有`火水雷风`4种，此外还有挖矿类（如钟离）以及纳西妲两个类型，可以考虑建立两支队伍`钟纳水雷`和`钟纳火风`，即可满足所有采集任务的需要。
 
-支持使用配置组`更多功能`——`日志分析`分析运行记录（参考了[mno](https://github.com/Bedrockx)大佬的写法）。
+配置好两支队伍后，脚本会在执行路线前计算所需角色，如果当前队伍不满足要求，会自动切换到另一支队伍。
+
+注意：队伍切换功能在联机模式下无法执行，因此联机采集时请先调整好队伍中的角色。
+
+# 进阶用法
+
+### 1. 不想每次订阅材料路线后都要在脚本设置中再勾选，怎么办？
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)